File: 2026_02/leetcode2026-02-01.py
from typing import List

class Solution:
    def minimumCost(self, nums: List[int]) -> int:
        res = nums[0]
        nums = sorted(nums[1:])
        return res+nums[0]+nums[1]

if __name__ == "__main__":
    s = Solution()
    nums = [1,2,3,12]
    print(s.minimumCost(nums))

File: leetcode2026/2026_02/leetcode2026-02-01.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

class Solution {
public:
    int minimumCost(vector<int>& nums) {
        int res = nums[0];
        nums.erase(nums.begin());
        sort(nums.begin(), nums.end());
        return res+nums[0]+nums[1];
    }
};

int main(){
    Solution s;
    vector<int> nums = {1,2,3,12};
    cout << s.minimumCost(nums) << endl;
}
